apps/angular/src/app/features/anime/routes.module.ts

```diff
@@ -0,0 +1,24 @@
+import { Routes, RouterModule } from '@angular/router';
+import { NgModule } from '@angular/core';
+
+import { AnimePageComponent } from './pages/anime/anime-page.component';
+import { DetailsPageComponent } from './pages/details/details-page.component';
+
+/** Routes object. */
+export const routes: Routes = [
+	{
+		path: '',
+		component: AnimePageComponent,
```

Comment:
The name of the "Anime" component in the "Anime" feature is unclear. For example, look at your "Details" component in the "Anime" feature, it is immediately clear what the component will do. I think for the "Anime" component it is worth using the name "List" or "Dashboard"